integrations/newrelic/newrelic_integration/ocean.py

```diff
@@ -0,0 +1,85 @@
+import fastapi
+from loguru import logger
+from port_ocean.context.ocean import ocean
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+from newrelic_integration.core.entities import EntitiesHandler
+from newrelic_integration.core.issues import IssuesHandler, IssueState
+from newrelic_integration.utils import (
+    get_port_resource_configuration_by_newrelic_entity_type,
+    get_port_resource_configuration_by_port_kind)
+
+
+@ocean.on_resync()
+async def resync_entities(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    logger.debug("Resyncing entities", kind=kind)
+    port_resource_configuration = await get_port_resource_configuration_by_port_kind(
+        kind
+    )
+    # mainly for issues newRelicAlert as it has different resync logic than entities
+    if not port_resource_configuration.get("selector", {}).get("entity_query_filter"):
+        logger.debug(
+            "Skipping resync for kind without entity_query_filter",
+            kind=kind,
+        )
+        return
+    else:
```

Comment:
Are you sure ocean supports this? You might need to return an empty list
Or yield one

Reply:
works 👍